src/muya/lib/parser/index.js

```diff
@@ -182,6 +183,23 @@ const tokenizerFac = (src, beginRules, inlineRules, pos = 0, top, labels) => {
     if (inChunk) continue
     // image
     const imageTo = inlineRules.image.exec(src)
+    if (imageTo && typeof imageTo[4] === 'string') {
+      const lastParenIndex = findClosingBracket(imageTo[4], '()')
+
+      if (lastParenIndex > -1) {
+        const len = 4 + imageTo[5].length + lastParenIndex + 1
+        imageTo[0] = imageTo[0].substring(0, len)
+        const originSrc = imageTo[4].substring(0, lastParenIndex)
+        const match = /(\\+)$/.exec(originSrc)
```

Comment:
What do we match here?

Reply:
Because this `originSrc` maybe have `\` at the end, it belongs to `imageTo[5]`, So we just do match here.